File: cpp/wejscie_wyjscie.cpp
/*
 * szablon.cpp
 */


#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
	int a, b;
    a = b = 0;
    
    cout << "Podaj pierwszą liczbę: ";
    cin >> a;
    cout << a;
    
    cout << endl << "Podaj drugą liczbę: ";
    cin >> b;
    cout << b;
    
    cout << endl << "Suma: " << a + b << endl;
    cout << endl << "Różnica: " << a - b << endl;
    cout << endl << "Iloraz: " << a / b << endl;
    
	return 0;
}

File: python/wejscie_wyjscie.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  wejscie_wyjscie.py
#  


def main(args):
    a = int(input("Podaj pierszą liczbę: "))
    print(a)
    
    b = int(input("Podaj drugą liczbę: "))
    print(b)
    
    print("Suma: ", a + b)
    
    print("Iloczyn: ", a * b)
    
    print("Iloraz: ", a / b)
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
